content/docs/using-github-directly/index.mdx

```diff
@@ -0,0 +1,303 @@
+8. Once the forking finishes, you will be taken to your own personal fork of DARIAH-CAMPUS. To make sure you are in the fork, check out that your user name (or the name of the organizational repository youo selected in step 7 above) is displayed in the upper left corner. ![](images/github_fork_check_your_name.png)
+9. Click on "Clone or download", then select "Open in Desktop". ![](images/github_clone_and_open_in_desktop.png)
+10. In GitHub Desktop, choose where you want to save the cloned DARIAH-CAMPUS fork on your computer. ![](images/github_desktop_saving_a_fork.png)
+11. The first you do this, it may take a couple of minutes, but don't despair, things will be quicker after the initial clone. Once you finish cloning your fork, you should see something like this: ![](images/github_desktop_result_of_cloned_fork.png)
+12. To get to the folder with DARIAH-CAMPUS on your computer, click on Open in Atom (or your preferred editor, if you already set it up).
+
+## Previewing DARIAH-Campus locally
+
+### Instructions for Mac-users
+
+1. Open up Terminal (usually found in ‘Utilities’ on a Mac).
+2. Go to `https://brew.sh/`
+3. Select the ‘Install Homebrew’ link and paste it into Terminal
+4. You will be prompted for a password in Terminal: enter your laptop password. The installation will begin.
+5. Next, type: "`brew install node`" into Terminal and hit ‘return’
+6. Then, you need to install ‘Yarn'. Type "`brew install yarn`" into Terminal, and hit ‘return’.
+
+Now that you have yarn installed on your system, you will be able to use it to install all the packages you need to run DARIAH-Campus locally on your computer. To do this:
+
+1. In Finder, go to the folder in which you cloned your DARIAH-Campus fork.
+2. In Terminal, type "`cd`" to change directory (also note, be sure to add a space after ‘cd’), then
+3. Drag and drop the folder from Finder into Terminal _on the same line as “cd “_. This will automatically write the full path (‘breadcrumb’) to the DARIAH-Campus folder in Terminal.
+4. Then type "`pwd`" (path to working directory) in Terminal and hit ‘return’ This will show the correct breadcrumb trail to the folder to make sure you have the right one.
+5. Type "`yarn install`" into Terminal (remember to hit ‘enter’). Installation may take a while if you haven’t installed this before. There is a progress bar on the bottom of Terminal that shows you how far along you are.
+
+Now that you have all the packages you need to preview DARIAH-Campus, you can start the server and build the site like this:
+
+1. In Terminal, go to your local DARIAH-Campus folder (see previous steps 1, and 3 in the previous section for how to do this)
+2. Type "`yarn develop`" (+ enter). This build the whole site locally and start a server to run it in. This may also take a little while.
```

Comment:
```suggestion
2. Type "`yarn dev`" (+ enter). This build the whole site locally and start a server to run it in. This may also take a little while.
```